translations/tiled_fr.ts

```diff
@@ -2595,7 +2596,7 @@ Voulez-vous le remplacer ?</translation>
     <message>
         <location line="-1223"/>
         <source>Ctrl+Shift+O</source>
-        <translation type="unfinished"></translation>
+        <translation>Ctrl+Màj+O</translation>
```

Comment:
Just as a reminder, you should not translate the names of keys in shortcuts, otherwise Qt will not be able to determine which keys are meant. Qt will take care of translating key names. I'll fix this one when cherry-picking the translation update to the 0.13 branch.